src/scrabble.js

```diff
@@ -1,5 +1,53 @@
-function scrabble() {
+let score = 0
 
+function findValues(word) {
```

Comment:
Nice use of using two functions. We struggled with that in some pair work done previously so good to see you've learnt from that.